modules/serverless-workflow-traffic-splitting.adoc

```diff
@@ -0,0 +1,43 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="traffic-splitting_{context}"]
+= Traffic splitting using Knative Client
+
+`kn` helps you control which revisions get routed traffic on your Knative service.
+
+Knative service allows for traffic mapping, which is the mapping of revisions of the service to an allocated portion of traffic. It offers the option to create unique URLs for particular revisions and has the ability to assign traffic to the latest revision.
+
+With every update to the configuration of the service, a new revision is created with the service route pointing all the traffic to the latest ready revision by default.
+
+You can change this behavior by defining which revision gets a portion of the traffic.
+
+
+.Procedure
+
+* Updating the traffic is done by using the `kn service update` command with the `--traffic` flag.
+
+[NOTE]
+====
+`--traffic RevisionName=Percent` uses the following syntax:
+
+* The `--traffic` flag requires two values separated by separated by an equals sign (`=`).
+
+* The `RevisionName` string refers to the name of the revision.
+
+* `Percent` integer denotes the traffic portion assigned to the revision.
+
+*  Use identifier `@latest` for the RevisionName to refer to the latest ready revision of the service. You can use this identifier only once with the `--traffic` flag.
+
+* If the `service update` command updates the configuration values for the service along with traffic flags, the `@latest` reference will point to the created revision to which the updates are applied.
+
+* `--traffic` flag can be specified multiple times and is valid only if the sum of the  `Percent` values in all flags totals 100.
+====
+
+* For example, to route 10% of traffic to your new revision before putting all traffic on, use the following command: 
```

Comment:
This looks a little odd to me. If it is only one step, that's when we use an unnumbered bullet. This makes it look like it's two steps, but it really just is giving an example.

I'd suggest to follow the format you used for assigning/unassigning tags, where you have just one bullet, and give the example command, and then show a note w/ the details afterward.